include/wx/webview.h

```diff
@@ -144,6 +144,27 @@ class WXDLLIMPEXP_WEBVIEW wxWebViewHandler
     wxString m_virtualHost;
 };
 
+class WXDLLIMPEXP_WEBVIEW wxWebViewConfigurationImpl
+{
+public:
+    virtual ~wxWebViewConfigurationImpl() = default;
+    virtual void* GetNativeConfiguration() const { return nullptr; }
+};
+
+class WXDLLIMPEXP_WEBVIEW wxWebViewConfiguration
+{
+public:
+    explicit wxWebViewConfiguration(const wxString& backend, wxWebViewConfigurationImpl* impl)
+        : m_backend(backend), m_impl(impl) {}
+    virtual ~wxWebViewConfiguration() = default;
```

Comment:
This class is not meant to be inherited from and so its dtor should _not_ be virtual.